### ISSUE TITLE
Fails when parsing a Puppet file with Windows line endings on Linux.

### ISSUE THREAD

**Issue by Aethylred** [rodjek/puppet-lint#262]
A .pp file can pass validation, and pass puppet-lint on Windows, but will fail under Linux if there are Windows CRLF line endings.

puppet-lint mistakenly reports these as a syntax error.

``` shell
$ bundle exec rake validate && bundle exec rake lint && bundle exec rake spec SPEC_OPTS='--format documentation'
puppet parser validate --noop manifests/gemrc.pp
puppet parser validate --noop manifests/init.pp
puppet parser validate --noop manifests/dev.pp
puppet parser validate --noop manifests/config.pp
puppet parser validate --noop manifests/params.pp
ruby -c spec/classes/config_spec.rb
Syntax OK
ruby -c spec/classes/gemrc_spec.rb
Syntax OK
ruby -c spec/classes/init_spec.rb
Syntax OK
ruby -c spec/spec_helper_system.rb
Syntax OK
ruby -c spec/spec_helper.rb
Syntax OK
manifests/init.pp - ERROR: Syntax error (try running `puppet parser validate <file>`) on line 68
rake aborted!
/home/travis/.rvm/gems/ruby-1.8.7-p374/gems/puppet-lint-0.3.2/lib/puppet-lint/tasks/puppet-lint.rb:25:in `initialize'
/home/travis/.rvm/gems/ruby-1.8.7-p374/gems/puppet-lint-0.3.2/lib/puppet-lint/tasks/puppet-lint.rb:13:in `send'
/home/travis/.rvm/gems/ruby-1.8.7-p374/gems/puppet-lint-0.3.2/lib/puppet-lint/tasks/puppet-lint.rb:13:in `initialize'
Tasks: TOP => lint
(See full trace by running task with --trace)
The command "bundle exec rake lint" exited with 1.
```
